packages/api-providers/src/web3/vanchor-deposit.ts

```diff
@@ -103,15 +112,15 @@ export class Web3VAnchorDeposit extends VAnchorDeposit<WebbWeb3Provider, Deposit
       exponentiation: '5',
       hashFunction: 'Poseidon',
       protocol: 'vanchor',
-      secrets: `${toFixedHex(destination, 6).substring(2)}:${BigNumber.from(
-        depositOutputUtxo.amount
-      ).toHexString()}:${depositOutputUtxo.keypair.pubkey.toHexString()}:${BigNumber.from(
-        depositOutputUtxo.blinding
-      ).toHexString()}`,
+      secrets:
+        `${toFixedHex(destination, 8).substring(2)}:` +
+        `${toFixedHex(depositOutputUtxo.amount).substring(2)}:` +
+        `${toFixedHex(keypair.privkey).substring(2)}:` +
+        `${depositOutputUtxo.blinding}`,
       sourceChain: sourceChainId.toString(),
```

Comment:
Can we set those pieces of data into their own variables and use the string interpolation instead of concatenating them we the `+`, or just use the data directly as no need for redundant interpolation `${string}`
another recommendation is to use an array
const secrets = [
toFixedHex(destination, 8).substring(2),// Dest chain
toFixedHex(depositOutputUtxo.amount).substring(2) // amount
...
].join(':')